run-tests.py

```diff
@@ -38,7 +38,7 @@ def cleanup(out):
         'has_attribute.cpp',
         'header_lookup1.c', # missing include <stddef.h>
         'line-directive-output.c',
-        'macro_paste_hashhash.c',
+        #  'macro_paste_hashhash.c',
```

Comment:
can you explain what happened here. do you cover everything with your test cases?

Reply:
ah sorry I misread as if you removed something.. but if you can comment out this line that is good news!!